docs/release-notes/mongoid-7.5.txt

```diff
@@ -0,0 +1,46 @@
+***********
+Mongoid 7.5
+***********
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+This page describes significant changes and improvements in Mongoid 7.5.
+The complete list of releases is available `on GitHub
+<https://github.com/mongodb/mongoid/releases>`_ and `in JIRA
+<https://jira.mongodb.org/projects/MONGOID?selectedItem=com.atlassian.jira.jira-projects-plugin:release-page>`_;
+please consult GitHub releases for detailed release notes and JIRA for
+the complete list of issues fixed in each release, including bug fixes.
+
+
+Order of Callbacks Invocation
+-----------------------------
+
+**Breaking change:** Mongoid 7.5 changes order of _create and _save callbacks
+invocation for documents with associations.
+
+Referenced associations (``has_one`` and ``has_many``):
+
+* Parent, :before_save
+* Parent, :around_save_open
+* Parent, :before_create
+* --> Parent persisted in MongoDB <--
+* Parent, :around_create_open
+* Child, :before_save
+* Child, :around_save_open
+* Child, :before_create
+* --> Child persisted in MongoDB <--
+* Child, :around_create_open
+* Child, :around_create_close
+* Child, :after_create
+* Child, :around_save_close
+* Child, :after_save
+* Parent, :around_create_close
+* Parent, :after_create
+* Parent, :around_save_close
+* Parent, :after_save
```

Comment:
Can you also add the 7.4 order please like we are doing in the release notes elsewhere (i.e. after showing what happens in the new version, we also show what was happening in the previous version so tht users can easily see the change/compare behavior).

Reply:
Done. I used a table to make it easier to see differences